### PR TITLE
Add coveralls integration

### PR DESCRIPTION
**Pull request by dalpo**
I found another interesting service about code coverage: https://coveralls.io/

Let me know what do you think. So finally we could have our coverage badge! :)

### CODE REVIEW

**Comment by potomak** [comment]
@dalpo we need to figure out how to remove those annoying comments!

**Comment by potomak** [comment]
@dalpo can you do something to fix the build so we can close this?

File: README.md [hunk 1]
```diff
@@ -88,6 +88,5 @@ Tomatoes is released under the [MIT license](https://raw.github.com/tomatoes-app
 
 [![Build Status](https://travis-ci.org/tomatoes-app/tomatoes.svg?branch=master)](https://travis-ci.org/tomatoes-app/tomatoes)
 [![Code Climate](https://codeclimate.com/github/tomatoes-app/tomatoes/badges/gpa.svg)](https://codeclimate.com/github/tomatoes-app/tomatoes)
-[![Test
-Coverage](https://codeclimate.com/github/tomatoes-app/tomatoes/badges/coverage.svg)](https://codeclimate.com/github/tomatoes-app/tomatoes/coverage)
+[![Coverage Status](https://coveralls.io/repos/github/tomatoes-app/tomatoes/badge.svg?branch=coveralls)](https://coveralls.io/github/tomatoes-app/tomatoes?branch=coveralls)
```
Comment (potomak) [inline README.md]: Do we need the `branch=coveralls` param?

Comment (dalpo) [inline README.md]: I think so, I copied it from the coveralls documentation and without it doesn't seem to work. :thinking:

**Pull request merged**